examples/vlan_learning.c

```diff
@@ -31,6 +36,13 @@ int handle_phys2virt(struct __sk_buff *skb) {
       // relearn when mac moves ifindex
       if (out_leaf->out_ifindex != skb->ifindex)
         out_leaf->out_ifindex = skb->ifindex;
+      // relearn when vlan_tci/vlan_proto changes
+      if (out_leaf->vlan_tci != skb->vlan_tci)
```

Comment:
Yes, we can do. I thought this before. But finally I just want to conform
to the existing style.
Let me change that.

On Wed, Jul 22, 2015 at 10:59 AM, Brenden Blanco <notifications@github.com>
wrote:

> In examples/vlan_learning.c
> <https://github.com/iovisor/bcc/pull/90#discussion_r35243097>:
>
> > @@ -31,6 +36,13 @@ int handle_phys2virt(struct __sk_buff *skb) {
> >        // relearn when mac moves ifindex
> >        if (out_leaf->out_ifindex != skb->ifindex)
> >          out_leaf->out_ifindex = skb->ifindex;
> > +      // relearn when vlan_tci/vlan_proto changes
> > +      if (out_leaf->vlan_tci != skb->vlan_tci)
>
> Can be assignment without conditional, no? Same for above line. Fewer
> branches.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iovisor/bcc/pull/90/files#r35243097>.
>